apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-roles/user-roles.component.html

```diff
@@ -1,8 +1,341 @@
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-facility-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.FACILITYOBSERVER' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.FACILITYOBSERVER_FACILITIES' | translate}}
+      <app-facility-select-table
+        [displayedColumns]="['id', 'name']"
+        [facilities]="facilities">
+      </app-facility-select-table>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getVos(roles.get('TRUSTEDFACILITYADMIN').get('Vo'))" *ngIf="isTrustedFacilityAdmin">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-facility-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.TRUSTEDFACILITYADMIN' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      <div>
+        {{'ROLES.TRUSTEDFACILITYADMIN_VOS' | translate}}
+        <perun-web-apps-vo-select-table
+          [displayedColumns]="['id', 'name', 'shortName']"
+          [vos]="vos">
+        </perun-web-apps-vo-select-table>
+      </div>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getVos(roles.get('TOPGROUPCREATOR').get('Vo'))" *ngIf="isTopGroupCreator">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-group-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.TOPGROUPCREATOR' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.TOPGROUPCREATOR_VOS' | translate}}
+      <perun-web-apps-vo-select-table
+        [displayedColumns]="['id', 'name', 'shortName']"
+        [vos]="vos">
+      </perun-web-apps-vo-select-table>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getMembers()" *ngIf="hasSponsorship">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-user-dark' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.SPONSORSHIP' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.SPONSORSHIP_MEMBERS' | translate}}
+      <perun-web-apps-members-list
+        [members]="members"
+        [hideColumns]="['checkbox', 'email', 'logins', 'status']"
+      ></perun-web-apps-members-list>
+    </div>
+  </mat-expansion-panel>
```

Comment:
when I want to show myself a Sponsorship card an error occurs
![image](https://user-images.githubusercontent.com/32871167/104293139-f6bbff80-54bd-11eb-8819-d70170a8f808.png)